reflex/state.py

```diff
@@ -1812,6 +1814,20 @@ class State(BaseState):
     # The hydrated bool.
     is_hydrated: bool = False
 
+    def handle_frontend_exception(self, stack: str) -> None:
```

Comment:
this should be moved into its own substate like the other built in functionality defined below.

having it here means that any frontend exception will trigger a load of the entire State tree with all substates. if it's in its own substate, then only that substate will get loaded